src/WebJobs.Script.Analyzers/WebJobsAttributeAnalyzer.cs

```diff
@@ -0,0 +1,126 @@
+﻿using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Azure.Functions.Analyzers
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class WebJobsAttributeAnalyzer : DiagnosticAnalyzer
+    {
+        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get { return ImmutableArray.Create(DiagnosticDescriptors.IllegalFunctionName); } }
+
+        public static void VerifyWebJobsLoaded()
+        {
+            // Check if WebJobs types can be loaded
+            var jobHost = new JobHost(new OptionsWrapper<JobHostOptions>(new JobHostOptions()), null);
```

Comment:
If we're not going to use the variable
```suggestion
            _ = new JobHost(new OptionsWrapper<JobHostOptions>(new JobHostOptions()), null);
```

Reply:
I do not think this `VerifyWebJobsLoaded` method is needed in this analyzer PR anymore. Based on our last sync, we decided to keep only the code needed for the analyzer for function name validation. Feel free to correct me @kashimiz

Reply:
True; removed.